algorithms/linfa-bayes/src/hyperparams.rs

```diff
@@ -5,7 +5,7 @@ use std::marker::PhantomData;
 /// A verified hyper-parameter set ready for the estimation of a [Gaussian Naive Bayes model](crate::gaussian_nb::GaussianNb).
 ///
 /// See [`GaussianNb`](crate::gaussian_nb::GaussianNb) for information on the model and [`GaussianNbParams`](crate::hyperparams::GaussianNbParams) for information on hyperparameters.
-#[derive(Debug)]
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
```

Comment:
`Copy` impl might bite us in the future, when we decide to add new fields to this struct.